src/installers.ts

```diff
@@ -77,20 +72,56 @@ const REDSCRIPT_FILE_EXT = ".reds";
  */
 const LUA_FILE_EXT = ".lua";
 
+const PRIORITY_STARTING_NUMBER = 30; // Why? Fomod is 20, then.. who knows? Don't go over 99
+
 // Types
 
 export enum InstallerType {
+  CET = "CET", // #13
+  Redscript = "Redscript", // #27
+  Red4Ext = "Red4ext", // #5
+  TweakDB = "TweakDB", // #6
+  AXL = "AXL", // #28
+  INI = "INI", // #29
+  Config = "Config", // #30
+  Reshade = "Reshade", // #8
+  LUT = "LUT", // #31
+  CoreCET = "Core/CET", // #32
+  CoreRedscript = "Core/Redscript", // #32
+  CoreRed4ext = "Core/Red4ext", // #32
+  CoreCSVMerge = "Core/CSVMerge", // #32
   ArchiveOnly = "ArchiveOnly",
   Other = "Other",
+  NotSupported = "Trying to install something not supported",
 }
 export interface Installer {
   type: InstallerType;
   id: string;
-  priority: number;
   testSupported: Vortex.TestSupported;
   install: Vortex.InstallFunc;
 }
+export interface InstallerWithPriority extends Installer {
+  priority: number;
+}
+
+// testSupported that always fails
+//
+export const notSupportedModType: Vortex.TestSupported = (
+  _files: string[],
+  _gameId: string,
+): Promise<Vortex.ISupportedResult> =>
+  Promise.resolve({ supported: false, requiredFiles: [] });
```

Comment:
I guess we don't need to even bother testing the game if this TestSupported always resolves to not be used.